agent/Makefile

```diff
@@ -237,7 +237,7 @@ install: install-destdir install-util-scripts install-bench-scripts install-tool
 	${COPY} profile ${DESTDIR}
 	${INSTALL} ${INSTALLOPTS} ${STOCKPILEDIR}
 	${COPY} stockpile ${DESTDIR}
-	${COPY} ../lib/configtools ${LIBDIR}
+	if [ -d ../lib/configtools ] ;then ${COPY} ../lib/configtools ${LIBDIR} ;fi
```

Comment:
Why is this necessary?  Don't we always have `../lib/configtools`?

Reply:
No - the `make install` that the rpm/Makefile does, copies (parts of) the agent/ subdir to some temp directory and then moves the lib/configtool directory from the *top* directory into the agent/lib sudbir in the copy. That's what ends up in the tarball and that's what the spec file has to deal with. So when the specfile does a `make install` to install the agent/ dir into the BUILDROOT, there is no ../lib/configtools any longer.

Reply:
Removed the need to invoke `make install` in the RPM spec file.